src/metabase/driver/hive_like.clj

```diff
@@ -0,0 +1,173 @@
+(ns metabase.driver.hive-like
+  (:require
+   [clojure.java.jdbc :as jdbc]
+   (clojure [set :as set]
+            [string :as s])
+   [clojure.tools.logging :as log]
+   (honeysql [core :as hsql]
+             [helpers :as h])
+   [metabase.db.spec :as dbspec]
+   [metabase.driver :as driver]
+   [metabase.driver.generic-sql :as sql]
+   [metabase.util :as u]
+   [metabase.util.honeysql-extensions :as hx]
+   [metabase.query-processor.util :as qputil]
```

Comment:
these are almost all in perfect alphabetical order (thanks!) except for this query-processor entry https://github.com/metabase/metabase/wiki/Metabase-Clojure-Style-Guide#keep-requires-ordered

Reply:
Done